crud-module/templates/client/controllers/_.client.controller.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
```

Comment:
This file should be changed once we have an example module with the new style guide in the main repo